apps/charterafrica/src/payload/fields/ecosystem/AirtableTableSelect.js

```diff
@@ -0,0 +1,44 @@
+import {
+  Select,
+  getSiblingData,
+  useAllFormFields,
+} from "payload/components/forms";
+import { select } from "payload/dist/fields/validations";
+import { createElement } from "react";
+import useSWR from "swr";
+
+import fetchJson from "../../../utils/fetchJson";
+
+const fetcher = (url) => fetchJson.get(url);
+const getOptions = async (baseId) => {
+  const url = `/api/v1/resources/ecosystem/proxy${baseId}/tables`;
+  const { bases } = await fetcher(url);
```

Comment:
```suggestion
const getOptions = async (baseId) => {
  const url = `/api/v1/resources/ecosystem/proxy${baseId}/tables`;
  const { bases } = await fetchJson.get(url);
```
Not sure fetcher adds anything here.